### PR TITLE
Add NLP distillation show case.

### PR DESCRIPTION
**Pull request by gongweibao**
非常感谢您给EDL项目提交PR!  
在提交PR之前，请帮忙回答一下问题以帮助我们判断PR的意图：  
- 这个PR要解决什么问题？
- 有没有对应的ISSUE？ 
  Fix
- 这个PR里边有没有要注意的地方？
- 这个PR改变了用户接口？
- 其他的说明？

You are welcome to submmit PR for EDL.   
Before this, would you like to answer some questions to help others to get what the PR does? 
- What this PR does / why we need it?
- Which issue(s) this PR fixes? 
  Fix
- Special notes for your reviewer:
- Does this PR introduce a user-facing change?
- Additional documentation?

### CODE REVIEW

File: example/distill/nlp/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# ERNIE distillation
+
+## Dataset
```
Comment (gavin1332) [inline example/distill/nlp/README.md]: description is required to make this task clear, not only the dataset. That is what this task is.

File: example/distill/nlp/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Chinese sentiment dataset
+
+## Local test
```
Comment (gavin1332) [inline example/distill/nlp/README.md]: where is the local test?

File: example/distill/nlp/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Finetune or downlod finetuned model
```
Comment (gavin1332) [inline example/distill/nlp/README.md]: where is the content?

File: example/distill/nlp/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+### Download dataset
+```
+wget https://paddle-edl.bj.bcebos.com/distillation/chnsenticorp/data.tgz
+tar -xzvf data.tgz
+```
+
+### Finetune
```
Comment (gavin1332) [inline example/distill/nlp/README.md]: finetuning is against pretraining. In distillation, it is the `teacher model` not the `finetuned model`.

File: example/distill/nlp/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+```
+nohup python -u ./fine_tune.py > finetune.log 2>&1 &
+```
+
+When the finetune job completes, the directory `ernie_senti_server` and `ernie_senti_client` will be generated.
+ 
+### Download finetuned model
```
Comment (gavin1332) [inline example/distill/nlp/README.md]: Download the teacher model

Comment (gavin1332) [reply example/distill/nlp/README.md]: you would like to tell user how to distill the knowledge of a teacher to a student in this example, so you'd better not involve new concept about finetuning.

**Review by gavin1332 — CHANGES_REQUESTED** [review]

File: example/distill/nlp/README.md (hunk 2 above)
Comment (gongweibao) [reply example/distill/nlp/README.md]: Done.

File: example/distill/nlp/README.md (hunk 3 above)
Comment (gongweibao) [reply example/distill/nlp/README.md]: Done.

File: example/distill/nlp/README.md (hunk 5 above)
Comment (gongweibao) [reply example/distill/nlp/README.md]: Done.

File: example/distill/nlp/README.md (hunk 4 above)
Comment (gongweibao) [reply example/distill/nlp/README.md]: Done.

File: example/distill/nlp/README.md (hunk 1 above)
Comment (gongweibao) [reply example/distill/nlp/README.md]: Done.

**Review by gavin1332 — APPROVED** [review]
LGTM

**Pull request merged**